src/pages/addCardNicknamePage.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useRef, useState } from "react";
+import { useLocation, useNavigate } from "react-router-dom";
+import styled from "styled-components";
+import { InputBox } from "../components";
+import { Card } from "../components/@common/card/card";
+import { Input } from "../components/@common/input/Input";
+import { InputGroup } from "../components/@common/input/inputGroup";
+import { PATH } from "../constants/path";
+import { setData } from "../utils/localStorage";
+
+export function AddCardNicknamePage() {
+  const { state } = useLocation();
+  const navigate = useNavigate();
+  const nicknameInput = useRef<HTMLInputElement>(null);
+
+  function completeInputNickname() {
+    const userCardInfo = { ...state };
+    if (nicknameInput.current !== null) {
+      userCardInfo["nickname"] = nicknameInput.current.value;
```

Comment:
```suggestion
      userCardInfo.nickname = nicknameInput.current.value;
```

보통 이렇게 더 많이 쓰는거 같아요~ 
자주 computed property name을 같은데, 뭔가 네이브만의 컨벤션이 있나요?

Reply:
아하...!! 컨벤션안 아니었고 뭐가 더 좋은지 몰라서 그냥 명시적으로 문자열을 넣어주려고 그랬어요..!! 알려주셔서 감사합니다!!!!👍

Reply:
리액트를 사용하게 되면 사실 객체의 속성값을 하나만 딱 저렇게 바꿀일이 거의 없습니다.
아래와 같이 새로운 객체를 생성하는게 일반적이니까요 ㅎㅎ

```ts
  const newUserCardInfo = { ...userCardInfo, nickname: nicknameInput.current.value };
```